packages/playground/templates/playground-page.php

```diff
@@ -6,25 +6,25 @@
 ?>
 <div id="wp-playground-wrapper">
     <div id="wp-playground-toolbar">
-        <span>
+        <p>
             <?php
             echo esc_attr(
                 sprintf(
                     // translators: %s: Site name.
                     __(
-                        'WordPress Playground preview for %s',
+                        'WordPress Playground preview of %s',
                         'playground'
                     ),
                     get_bloginfo('name')
                 )
             );
             ?>
-        </span>
+        </p>
         <a href="<?php echo esc_url(admin_url()); ?>" id="goBack">
```

Comment:
```suggestion
        <a href="<?php echo esc_url(admin_url()); ?>" id="goBack" class="button">
```

Reply:
What if we used the default button style here and just set the background to transparent? 
This would make the size more consistent with the rest of the UI.

Reply:
Great idea!

But the current link color (`#2271b1`) is still not passing AAA.
Chrome suggested `#1a5687` (what I used in the PR).

Reply:
> But the current link color (#2271b1) is still not passing AAA.

Probably because of the background. Would you have any alternative suggestions for how to indicate to the user that they are not on the real site anymore? The bar seems like an overall bad solution and maybe a border around the screen or something else would be enough.

Reply:
If enabling the Playground header isn't an option (due to functionality limitations—see [my questions below](https://github.com/WordPress/playground-tools/pull/228#discussion_r1566206071)), how about adding an admin notice? Something like this:
Display Admin Notice | [Preview](https://playground.wordpress.net/?blueprint-url=https://raw.githubusercontent.com/adamziel/blueprints/trunk/blueprints/admin-notice/blueprint.json) | [Source](https://github.com/adamziel/blueprints/blob/trunk/blueprints/admin-notice/blueprint.json)
-- | -- | --

Reply:
I'm not sure if a notice is the best solution. A topical admin area is already flooded with notices and they are frequently ignored. I think that the Playground header is the best option.